src/mudlet.qrc

```diff
@@ -13,6 +13,7 @@
         <file>icons/aardwolf_mud.png</file>
         <file>icons/accessories-text-editor.png</file>
         <file>icons/achaea_120_30.png</file>
+        <file>icons/abandonedrealms_icon.jpg</file>
```

Comment:
Do we have a large resolution `.png` file for this MUD, `.jpg`s, being lossy-compressed may not be the best choice if we can get a (possibly loss-less compressed) `.png` version?

Reply:
![ar-browser-icon](https://github.com/Mudlet/Mudlet/assets/110988/db851714-b17f-48a3-8419-8f8d1263f8cc)
![AR-Old-Stones 480x60](https://github.com/Mudlet/Mudlet/assets/110988/89b6a2f9-b44a-4d24-bdac-ef8427920e8a)
![AR-Old-Stones 140x50](https://github.com/Mudlet/Mudlet/assets/110988/5abe3785-fea3-47d4-acc6-3ece6104e556)